api-report/container-utils.api.md

```diff
@@ -20,21 +21,25 @@ export function CreateContainerError(originalError: any, props?: ITelemetryPrope
 export const CreateProcessingError: typeof DataProcessingError.wrapIfUnrecognized;
 
 // @public (undocumented)
-export class DataCorruptionError extends LoggingError implements IErrorBase {
+export class DataCorruptionError extends LoggingError implements IErrorBase, IFluidErrorBase {
     constructor(errorMessage: string, props: ITelemetryProperties);
     // (undocumented)
     readonly canRetry = false;
     // (undocumented)
     readonly errorType = ContainerErrorType.dataCorruptionError;
+    // (undocumented)
+    readonly fluidErrorCode = "TBD";
 }
 
 // @public (undocumented)
-export class DataProcessingError extends LoggingError implements IErrorBase {
+export class DataProcessingError extends LoggingError implements IErrorBase, IFluidErrorBase {
     constructor(errorMessage: string, props?: ITelemetryProperties);
     // (undocumented)
     readonly canRetry = false;
     // (undocumented)
     readonly errorType = ContainerErrorType.dataProcessingError;
+    // (undocumented)
+    readonly fluidErrorCode = "TBD";
```

Comment:
At a minimun i think this should be errorType, TBD is useless

Reply:
TBD will give huge buckets, error type will give better buckets, real error code will be the best. i also worry this will override all the errorCodeIfNone, as in this case there is one